### DISCUSSION TITLE
CVE-2023-5217

### DISCUSSION THREAD

**Discussion by PJ-Resp3ct** [flutter-webrtc/flutter-webrtc#1498] [General]
Hello,

As I can see the latest flutter_webrtc v.0.9.47 ships a libwebrtc.so version that depends on WebM/libvpx v.1.13.0-243-g27171320f. Unless I'm mistaken this version looks vulnerable to a severe vulnerability CVE-2023-5217 as the fix is in WebM/libvpx v1.13.1. 

What are your plans in regards to updating to latest WebRTC library (M119)?

Thanks in advance.

**Comment by davidliu** [comment]
The underlying webrtc has been patched to fix this vulnerability.

See the PR here: https://github.com/webrtc-sdk/webrtc/pull/98

Unfortunately, [this version string is a bit tricky to update](https://github.com/webrtc-sdk/webrtc/pull/98#issuecomment-1891694658) and does not accurately reflect the code.
